app/models/report.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Report < ApplicationRecord
+  has_many :comments, as: :commentable, dependent: :destroy
+  belongs_to :user
```

Comment:
```suggestion
  belongs_to :user
  has_many :comments, as: :commentable, dependent: :destroy
```

belongs_to → has_many の順番で書くことが多いですね

Reply:
なるほど...参考になります